.github/workflows/benchmark.yml

```diff
@@ -2,9 +2,7 @@ name: Run Best benchmark
 
 # We could run on every PR, but there is not much need right now.
 # By running on every push to master, we get historical Best data for the master branch.
-on:
-  push:
-    branches: [ master ]
+on: push
```

Comment:
I assume this is just a temporary change to test, right?

Reply:
@nolanlawson Yes, just testing things to make sure everything is working.